experiment/gerrit-onboarder/onboarder.go

```diff
@@ -0,0 +1,357 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/cmd/generic-autobumper/bumper"
+)
+
+const (
+	UUID                = 0
+	GROUP_NAME          = 1
+	GROUPS_FILE         = "groups"
+	PROJECT_CONFIG_FILE = "project.config"
+
+	ACCESS_HEADER            = "[access \"refs/*\"]"
+	PROW_READ_ACCESS_FORMAT  = "read = group %s"
+	PROW_LABEL_ACCESS_FORMAT = "label-Verified = -1..+1 group %s"
+	LABEL_HEADER             = "[label \"Verified\"]"
+)
+
+type Options struct {
+	Host       string
+	Repo       string
+	UUID       string
+	GroupName  string
+	BranchName string
+}
+
+func getAllLabelLines() []string {
+	return []string{
+		"function = MaxWithBlock",
+		"value = -1 Failed",
+		"value = 0 No score",
+		"value = +1 Verified",
+		"copyAllScoresIfNoCodeChange = true",
+		"defaultValue = 0",
+	}
+}
+
+func parseAndValidateOptions() (*Options, error) {
+	var o Options
+	flag.StringVar(&o.Host, "host", "", "The gerrit host.")
+	flag.StringVar(&o.Repo, "repo", "", "The gerrit Repo.")
+	flag.StringVar(&o.UUID, "uuid", "", "The UUID to be added to the file.")
+	flag.StringVar(&o.GroupName, "group", "", "The corresponding group name for the UUID.")
+	flag.StringVar(&o.BranchName, "branch", "", "The name of the branch where refs/meta/config will live")
```

Comment:
branch is not a gerrit thing imo, it's a local ref instead of remote. can we use a random string here instead of asking user to supply?